blynklib_mp.py

```diff
@@ -203,6 +218,13 @@ def _get_socket(self):
             self._socket = socket.socket()
             self._socket.connect(socket.getaddrinfo(self.server, self.port)[0][-1])
             self._set_socket_timeout(self.SOCK_TIMEOUT)
+            if self.port == 443 or self.port == 8443:
```

Comment:
We can't use ports as ssl definition
cause users can use custom local blynk server
If this will be ssl oriented file we can modify 
all logic to use ssl mode only and remove 
non ssl blocks.
so here will no such if with ports numbers

Reply:
Yes I see the issue. Let see what we end up doing about the file split and then we can choose what we do here.